cli/tests/default/unit/test_engine_type.py

```diff
@@ -2,79 +2,127 @@
 
 from semgrep.app.scans import ScanHandler
 from semgrep.engine import EngineType as ET
+from semgrep.error import SemgrepError
 from semgrep.meta import GitMeta
 
 
 @pytest.mark.quick
+@pytest.mark.parametrize("is_supply_chain_only", [True, False])
+@pytest.mark.parametrize("is_secrets_scan", [True, False])
 @pytest.mark.parametrize(
-    ("is_cloud_flag_on", "is_ci_scan_full", "enable_pro_diff", "requested", "expected"),
+    "engine_flag", [None, ET.OSS, ET.PRO_LANG, ET.PRO_INTRAFILE, ET.PRO_INTERFILE]
+)
+@pytest.mark.parametrize(
+    (
+        "logged_in",
+        "is_interfile_flag_on",
+        "is_ci_scan_full",
```

Comment:
This being `None` means that it's `semgrep scan` ?

Reply:
I added a comment about both of the flags that can be None. Hopefully that + renaming helps! If you have better suggestions for names let me know